pages/devs/topic-creators/how-to-create-topic.mdx

```diff
@@ -38,6 +38,10 @@ type MsgCreateNewTopic struct {
   AllowNegative    bool     `json:"allow_negative,omitempty"`
   // the numerical precision at which the network should distinguish differences in the logarithm of the loss
   Epsilon          github_com_allora_network_allora_chain_math.Dec  `json:"epsilon"`
+  MeritSortitionAlpha github_com_allora_network_allora_chain_math.Dec  `json:"merit_sortition_alpha"`
```

Comment:
There are comments in the other fields, seems proper to add them here too.

Reply:
MeritSortitionAlpha: 
// alpha parameter to weight previous score EMA importance in filtering the active set of inferers, forecasters, and reputers

ActiveXQuantile
// marks the quantile of lower Xs by score on the active set that may be replaced by new Xs each epoch